src/accounts/solana.ts

```diff
@@ -1,9 +1,11 @@
 import { Account } from "./account";
 import { BaseMessage, Chain } from "../messages/message";
 import { GetVerificationBuffer } from "../messages";
-import * as solanajs from "@solana/web3.js";
-import nacl from "tweetnacl";
 import base58 from "bs58";
+import nacl from "tweetnacl";
+import * as solanajs from "@solana/web3.js";
+import * as ecies25519 from "ecies-25519";
+import { Buffer } from "buffer";
```

Comment:
You import the `Buffer` polyfill but this it isn't done anywhere else in the code, this could maybe lead to unexpected behaviors.